src/components/ContinuationIn/ManualBusinessInfo.vue

```diff
@@ -320,114 +341,74 @@ export default class ManualBusinessInfo extends Mixins(CountriesProvincesMixin,
-      this.jurisdiction?.text.toLowerCase()
-    )
     if (jurisdiction?.group === 0) {
-      this.business.homeJurisdiction.country = JurisdictionLocation.CA
-      if (jurisdiction.value === JurisdictionLocation.FD) {
-        this.business.homeJurisdiction.region = 'FEDERAL'
-      } else {
-        this.business.homeJurisdiction.region = jurisdiction.value
-      }
+      // set property reactively (in case it was null)
+      this.$set(this.business, 'homeJurisdiction', {
+        country: JurisdictionLocation.CA,
+        region: (jurisdiction.value === JurisdictionLocation.FD) ? 'FEDERAL' : jurisdiction.value
+      })
     }
 
     if (jurisdiction?.group === 1) {
-      this.business.homeJurisdiction.country = JurisdictionLocation.US
-      this.business.homeJurisdiction.region = jurisdiction.value
+      // set property reactively (in case it was null)
+      this.$set(this.business, 'homeJurisdiction', {
+        country: JurisdictionLocation.US,
+        region: jurisdiction.value
+      })
     }
 
     if (jurisdiction?.group === 2) {
-      this.business.homeJurisdiction.country = jurisdiction.value
-      this.business.homeJurisdiction.region = ''
+      // set property reactively (in case it was null)
+      this.$set(this.business, 'homeJurisdiction', {
+        country: jurisdiction.value,
+        region: ''
+      })
     }
-
-    if (this.jurisdiction) {
-      this.setShowErrors(false)
-    }
-
-    this.jurisdictionErrorMessage = this.jurisdiction ? '' : 'Home jurisdiction is required'
-  }
-
-  get identifyingNumberRules (): Array<(v: string) => boolean | string> {
-    return [
-      v => !!v || 'Identifying Number is required'
-    ]
-  }
-
-  get businessNameRules (): Array<(v: string) => boolean | string> {
-    return [
-      v => !!v || 'Business Name is required'
-    ]
-  }
-
-  get incorporationDateRules (): Array<(v: string) => boolean | string> {
-    return [
-      v => !!v || 'Incorporation Date is required'
-    ]
-  }
-
-  @Watch('business', { deep: true })
-  private async onBusinessChanged (): Promise<void> {
-    this.setExistingBusinessInfo(this.business)
-    this.active = true
   }
 
+  /** Validates the form and other components. */
+  @Watch('active')
   @Watch('getShowErrors')
-  @Watch('business.homeJurisdiction')
-  @Watch('business.incorporationDate')
-  private onShowErrors (): void {
-    if (!this.jurisdiction && this.getShowErrors) {
-      this.jurisdictionErrorMessage = this.jurisdiction ? '' : 'Home jurisdiction is required'
-    }
-    if (this.jurisdiction && !this.isMrasJurisdiction && this.getShowErrors) {
-      this.$refs.manualFormRef.validate()
-      this.$refs.incorporationDateRef.validateForm()
+  private async onGetShowErrors (): Promise<void> {
+    if (this.active && this.getShowErrors) {
+      await this.$nextTick() // wait for form to finish rendering
+      this.$refs.manualFormRef?.validate()
+      this.$refs.incorporationDateRef?.validateForm()
```

Comment:
When the "active" and "getShowErrors" flags change and are both True, this code is executed.

You can see that I call the v-form validation to trigger validations on the Vuetify components, and I explicitly call our custom component's validation.

What caused me some grief here was needing to wait after "active" becomes true so that the template is rendered and the refs are initialized (otherwise they are undefined). It now works correctly and I can explain why (no magic).